rust/Makefile.am

```diff
@@ -128,8 +128,7 @@ if HAVE_BINDGEN
 		--default-enum-style rust \
 		--allowlist-type 'AppProto.*' \
 		--allowlist-function 'AppProto.*' \
-		--allowlist-type 'SC.*' \
-		--allowlist-function 'SC.*' \
+		--allowlist-item 'SC.*' \
```

Comment:
This will actually fail on the some versions of bindgen that otherwise will work, I think the bindgen on Ubuntu 22.04 maybe?

Reply:
adding `--allowlist-var 'SC.*'` then